waftools/openocd_swd_tigard.cfg

```diff
@@ -0,0 +1,12 @@
+adapter driver ftdi
+transport select swd
+ftdi vid_pid 0x0403 0x6010
+ftdi device_desc "Tigard V1.1"
+ftdi channel 1
+adapter speed 2000
+ftdi layout_init 0x0038 0x003b
+ftdi layout_signal SWD_EN -data 0
+#ftdi layout_signal nSRST -data 0x0020
```

Comment:
Intentionally left commented?

Reply:
Yes.  It doesn't seem to work right on my board, so I don't give openocd the opportunity to try it.